pyproject.toml

```diff
@@ -28,8 +28,11 @@ dependencies = [
   "dataclasses-json",
   "importlib_resources",
   "jsonschema",
-  "pygit2"
+  "pygit2",
+  "outpack_query_parser @ git+https://github.com/mrc-ide/outpack_server@python-bindings"
```

Comment:
The git url is temporary, until the python bindings get merged and we make a pip release for them.